app/backend/app.py

```diff
@@ -204,7 +205,7 @@ async def chat(request: Request):
     json_body = await request.json()
     approach = json_body.get("approach")
     try:
-        impl = chat_approaches.get(approach)
+        impl = chat_approaches.get(Approaches(int(approach)))
```

Comment:
Was the change to the enum from String to Int required for technical reason or purely aesthetic?

Reply:
Technically a better practice and easier to manage.  Simple refactor